.github/workflows/ci_docker.yml

```diff
@@ -13,6 +13,10 @@ on: # Trigger the workflow on push or pull request, but only for the master bran
       - ".github/workflows/*docker*.yml"
       - "setup.py"
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+#  cancel-in-progress: true
```

Comment:
Why is this active in `pytorch-lightning` but not in `metrics`?

Reply:
what do you mean by active?
 the screenshot is from this PR, when I created PR, had running jobs, and with update branch some as set were canceled

Reply:
I just meant that in PL the docker tests are going to be cancelled if they were already running while here not. Basically in PL you have `cancel-in-progress: true` and here `#  cancel-in-progress: true`.

Reply:
I see, yes we can do it too...
mainly it was that PL has much more and more complex dockers, here they are simpler :rabbit:

Reply:
```suggestion
  cancel-in-progress: true
```

Reply:
Oh I understand....is there any drawback in cancelling them? Because from what I see they take somewhat between 3 and 4 minutes.

Reply:
you do not see the result I tend to keep the quick ones and auto-cancel all mid and long runs